Source/Dafny/Plugin.cs

```diff
@@ -0,0 +1,82 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Microsoft.Dafny.Plugins;
+
+namespace Microsoft.Dafny;
+
+/// <summary>
+/// This class is internal to Dafny. It wraps an assembly and an extracted configuration from this assembly,
```

Comment:
> This class is internal to Dafny. 

What do you mean by that? Why not mark the visibility as internal then?

Reply:
I removed "is internal to Dafny. It wraps" since most classes in Dafny are never used outside of Dafny but could be in the future.